tests/src/main/python/TestDriver_Json.py

```diff
@@ -64,7 +65,7 @@ def setup_class(self):
         cohortEngine = baseDir+os.environ['COHORT_ENGINE']
         javabridge.start_vm(run_headless=True, class_path=javabridge.JARS + [cohortEngine, testWrapper]) # Start the JVM with modified classpath.
         engineWrapper = javabridge.JClassWrapper("com.ibm.cohort.engine.test.TestWrapper")() # Get an instance of the test wrapper.
-        engineWrapper.warm(baseDir+os.environ['DATA_FHIR_SERVER_DETAILS'],baseDir+os.environ['TERM_FHIR_SERVER_DETAILS'],libraries, "Test", "1235008") # Warm up the JV and submit a noise query.
+        engineWrapper.warm(baseDir+os.environ['DATA_FHIR_SERVER_DETAILS'],baseDir+os.environ['TERM_FHIR_SERVER_DETAILS'],libraries, testLibrary, "1235008") # Warm up the JV and submit a noise query.
```

Comment:
I'm not sure how much you care about Python style guidelines, but the lack of whitespace on certain lines in this file comes across as strange to me.

[codestyle.txt](https://github.com/Alvearie/quality-measure-and-cohort-service/files/5524787/codestyle.txt)
I ran this file through `pycodestyle` in case that interests you at all.